tracer/build/_build/Build.Shared.Steps.cs

```diff
@@ -154,6 +155,8 @@ partial class Build
 
                 // Copy binary to the temporal destination
                 CopyFile(sourceFile, destFile, FileExistsPolicy.Overwrite);
+                DeleteFile(sourceFile);
+                DeleteFile(NativeLoaderProject.Directory / "bin" / $"{NativeLoaderProject.Name}.static.a");
```

Comment:
Why? 🤔

Reply:
If CMAKE detects the static library then it doesn't build it again, but this static library belongs to the previous architecture. So I have to manually delete it, to produce a new static lib for the next architecture.

The native build process have two steps: Static (.a/.lib) + Dynamic (.dylib/.so/.dll)